vendor/github.com/aws/aws-sdk-go/service/ec2/api.go

```diff
@@ -70475,6 +70475,24 @@ const (
 	// InstanceTypeT22xlarge is a InstanceType enum value
 	InstanceTypeT22xlarge = "t2.2xlarge"
 
+	// InstanceTypeT3Micro is a InstanceType enum value
+	InstanceTypeT3Micro = "t3.micro"
```

Comment:
Are these changes you made yourself or are they from the upstream aws/aws-sdk-go package? If the former, we need to chat about how we are going to manage these patches as we typically do not change code in the vendor directory.  If the latter, we will mant to make sure we get go.mod and go.sum updated to account for what version these are in.

Reply:
I made those manually since I was changing the type to t3.micro.  I will revert this file.